test-jest/shape/Sector.spec.test.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { Surface, Sector } from '../../src';
+
+describe('<Sector />', () => {
+  test('Render 1 sector in simple Sector', () => {
+    render(
+      <Surface width={500} height={500}>
+        <Sector fill="#ff7902" cx={200} cy={300} innerRadius={150} outerRadius={200} endAngle={90} />
+      </Surface>,
+    );
+    const sectorEl = screen.getByRole('img');
```

Comment:
I feel like tests by `img` role might be flaky if we ever decide to remove that from the elements (since really the elements are not images...) but probably fine for the sake of the migration.

Reply:
As far as I know, all shapes drawn in recharts are svg elements. So when I test for shape, I think this code makes no problem.

Reply:
Thank for the review and for letting me contribute! 🙂